### PR TITLE
#154 - Add IT and slice  for npm unpublish

### PR DESCRIPTION
**Pull request by genryxy**
Part of #154 
Add IT and slice without realization  for `npm unpublish`

### CODE REVIEW

File: src/test/java/com/artipie/npm/NpmUnpublishIT.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+    /**
+     * Container.
+     */
+    private GenericContainer<?> cntn;
+
+    @BeforeEach
+    void setUp() throws Exception {
+        this.vertx = Vertx.vertx();
+        this.storage = new FileStorage(this.tmp);
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/npm/NpmUnpublishIT.java:92]: @genryxy I'd suggest to use `InMemory` storage here, I do not see any need in `FileStorage`

File: src/test/java/com/artipie/npm/NpmUnpublishIT.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        this.server.stop();
+        this.vertx.close();
+        this.cntn.stop();
+    }
+
+    @Test
+    void npmUnpublishWorks() throws Exception {
+        final String proj = "@hello/simple-npm-project";
+        new TestResource("storage")
+            .addFilesTo(this.storage, Key.ROOT);
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/npm/NpmUnpublishIT.java:120]: @genryxy seems like there is no need for line brake in this expression

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy thanks, just two minor suggestions

**Comment by genryxy** [comment]
@olenagerasimova thanks, used `InMemoryStorage`

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks, good to merge

**Comment by genryxy** [comment]
@g4s8 could you please check/merge

**Comment by g4s8** [comment]
@genryxy can you please check CI errors after rebase.

**Comment by genryxy** [comment]
@g4s8 fixed

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#167` is not assigned, can't get performer

**Pull request merged**